packages/webapp/src/webview/ui/components/HomeGrid/Bookmarks/Bookmarks.tsx

```diff
@@ -0,0 +1,96 @@
+            newBookmarks[bookmarkKey] = {
+                tree: bookmark.tree,
+                nodePath: bookmark.nodePath,
+                createdAt: new Date().toISOString()
+            };
+        }
+        actions.updateBookmark({ bookmarkKey, bookmarks: newBookmarks });
+    };
+
+    return (
+        <div>
+            <h3 className="guided-answer__home-grid__section__title">
+                <UIIcon iconName={UiIcons.StarActive} />
+                <span>Bookmarks</span>
+            </h3>
+            <FocusZone direction={FocusZoneDirection.bidirectional} isCircularNavigation={true}>
+                <ul className="guided-answer__home-grid__section__list" role="listbox">
+                    {Object.keys(localBookmarks).map((bookmarkKey) => {
+                        const bookmark = localBookmarks[bookmarkKey];
+                        const bookmarkTitle =
+                            bookmark.nodePath.length > 1
+                                ? `${bookmark.tree.TITLE} - ${bookmark.nodePath[bookmark.nodePath.length - 1].TITLE}`
+                                : bookmark.tree.TITLE;
+                        const isBookmarked = !!storedBookmarks[bookmarkKey];
+                        return (
+                            <li key={`tree-item-${bookmarkTitle}`} className="guided-answer__bookmark">
+                                <div className="guided-answer__bookmark__title">
+                                    <a id="goto-bookmark-button" onClick={(): void => goToBookmark(bookmark)}>
+                                        {bookmarkTitle}
+                                    </a>
+                                    <UIIconButton
+                                        id="toggle-bookmark-button"
+                                        title={
+                                            isBookmarked
+                                                ? i18next.t('REMOVE_FROM_BOOKMARKS')
+                                                : i18next.t('BOOKMARK_THIS_GUIDE')
+                                        }
+                                        iconProps={{ iconName: isBookmarked ? UiIcons.StarActive : UiIcons.Star }}
+                                        onClick={() => toggleBookmark(bookmark)}></UIIconButton>
+                                    {/* <button id="toggle-bookmark-button" onClick={(e) => toggleBookmark(e, bookmark)}>
+                                        {isBookmarked ? (
+                                            <VscStarFull
+                                                className="bookmark-icon-bookmarked"
+                                                title={i18next.t('REMOVE_FROM_BOOKMARKS')}
+                                            />
+                                        ) : (
+                                            <VscStarEmpty title={i18next.t('BOOKMARK_THIS_GUIDE')} />
+                                        )}
+                                    </button> */}
```

Comment:
Is there a reason for commented out code or is it a leftover?

Reply:
leftover, will remove